scripts/google/covid_mobility/covidmobility.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from random import random
+from os import remove
+import sys
+
+sys.path.insert(1, '../../../')
+import util.county_to_dcid as county_to_dcid
+import util.alpha2_to_dcid as alpha2_to_dcid
+import util.name_to_alpha2 as name_to_alpha2
+
+
+def skip_value(skip_pct) -> bool:
+    """Given the amount of cells you skip in percent, will return whether to skip that specific one."""
+    return random() <= skip_pct
+
+
+def covid_mobility(input_csv: str = None, less_output: bool = False, output_path='./output/covid_mobility_output.mcf') -> None:
+    if not input_csv:
+        raise Exception("You must input a csv file")
+
+    mobility_locations: dict = {}
+    data: list = []
+
+    # Remove any existing output file
+    remove(output_path)
+
+    with open(input_csv, 'r+') as f:
+        # Ignore the first row from csv, since it's the column titles.
+        rows: list = f.read().split("\n")[1:] # Read columns from input_csv
```

Comment:
Can we download the CSV in the script too? Then we can do the cleaning first.

Also, instead of having a full read and then separate by line, then calling `csv_row_to_obj`; you can use `csv.DictReader`, which would build the dict for you and you can directly iterate with `for row_dict in reader:` where reader is the DictReader. https://docs.python.org/3/library/csv.html#csv.DictReader

Reply:
done!